tests/python/test_percpu.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# Copyright (c) PLUMgrid, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+from bcc import BPF
+from time import sleep
+import unittest as ut
+
+class TestPercpu(ut.TestCase):
+
+    def test_u64(self):
+        test_prog1 = """
+        BPF_TABLE("pc_hash", u32, u64, stats, 1);
```

Comment:
Probably still would, the macro would probably just `##` append a different name. The rewriter needs a hint to be able to pick the map_type variable.